wandb/sdk/wandb_run.py

```diff
@@ -3199,20 +3219,23 @@ def _footer_multiple_runs_file_pusher_status_info(
                 ]
             )
             for response in poll_exit_responses
-            if response and response.file_counts
+            if response is not None and response.file_counts is not None
```

Comment:
is this your style or is this mypy enforcing this?

Reply:
mypy!